source/javascripts/controllers/awards_controller.coffee

```diff
@@ -12,6 +11,14 @@ EGA.controller "AwardsController", ($scope, $http, filterPipeline) ->
     countries: names: [], selected: []
     searchTerm: ''
 
+  $scope.categoryNames =
+    fire: 'Feuer'
+    water: 'Wasser'
+    air: 'Luft'
+    earth: 'Erde'
+    youth: 'Jugend'
+    other: 'Andere'
+
```

Comment:
just create a value object. get it out of the controller. we can refactor the value object later. just make a reasonable api.